test/suites/dev-tanssi-relay/xcm/test-export-expected--errors.ts

```diff
@@ -398,5 +398,447 @@ describeSuite({
                 expect(tokenTransferNonceAfter.toBigInt()).toBe(tokenTransferNonceBefore.toBigInt());
             },
         });
+
+        it({
+            id: "T05",
+            title: "Should fail exporting to a 32 byte address in ethereum",
+            test: async () => {
+                if (shouldSkipStarlightContainerExport) {
+                    console.log(`Skipping XCM tests for Starlight version ${specVersion}`);
+                    return;
+                }
+
+                const ethereumNetwork = { Ethereum: { chainId: TESTNET_ETHEREUM_NETWORK_ID } };
+
+                const xcmToExport = new XcmFragment({
+                    assets: [
+                        {
+                            multilocation: containerAsset,
+                            fungible: transferredBalance / 10n,
+                        },
+                    ],
+                    beneficiary: u8aToHex(random.addressRaw),
+                })
+                    .reserve_asset_deposited()
+                    .clear_origin()
+                    .buy_execution()
+                    .deposit_asset_v3()
+                    .set_topic();
+
+                const xcmMessage = new XcmFragment({
+                    assets: [
+                        {
+                            multilocation: {
+                                parents: 0,
+                                interior: { Here: null },
+                            },
+                            fungible: transferredBalance / 10n,
+                        },
+                    ],
+                    beneficiary: u8aToHex(random.addressRaw),
+                })
+                    .push_any({
+                        DescendOrigin: {
+                            X1: {
+                                Parachain: 2000,
+                            },
+                        },
+                    })
+                    .withdraw_asset()
+                    .buy_execution()
+                    .export_message(xcmToExport.instructions, ethereumNetwork, "Here")
+                    .as_v3();
+
+                const tokenTransferNonceBefore =
+                    await polkadotJs.query.ethereumOutboundQueue.nonce(tokenTransferChannel);
+
+                const executeMessageTx = polkadotJs.tx.xcmPallet.execute(xcmMessage, {
+                    refTime: 10000000000,
+                    proofSize: 1000000,
+                });
+
+                const { result } = await context.createBlock(
+                    polkadotJs.tx.sudo.sudo(executeMessageTx).signAsync(alice)
+                );
+                // sudo calls are always true
+                expect(result.successful).to.be.true;
+
+                const tokenTransferNonceAfter =
+                    await polkadotJs.query.ethereumOutboundQueue.nonce(tokenTransferChannel);
+
+                expect(tokenTransferNonceAfter.toBigInt()).toBe(tokenTransferNonceBefore.toBigInt());
```

Comment:
Previous tests also check for the emitted error, can you copy that code here? No need to test the exact error message, just that there is some failure.

Reply:
sure